src/lib/components/common/ui/floatbar.ts

```diff
@@ -1,43 +1,50 @@
-import {LitElement, html, css} from 'lit';
-import {customElement, property} from 'lit/decorators.js';
+import {html, css} from 'lit';
+import {property} from 'lit/decorators.js';
+import {FloatElement} from '../../custom';
+import {CustomElement} from '../../injectors';
+import {tooltip, tooltipStyles} from './tooltip';
 
-@customElement('float-bar')
-export class FloatBar extends LitElement {
+@CustomElement()
+export class FloatBar extends FloatElement {
     @property({type: Number}) float!: number;
     @property({type: Number}) minFloat = 0;
     @property({type: Number}) maxFloat = 1;
 
-    static styles = css`
-        .market-float-bar-container {
-            position: relative;
-            width: 100%;
-            height: 8px;
-            margin: 5px 0;
-        }
+    static styles = [
+        ...FloatElement.styles,
+        ...tooltipStyles,
```

Comment:
I'm fine with importing `tooltipStyles` inside of `FloatElement` and also importing `tooltip` in `FloatElement` so any element can just use the directive.